daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/command/Command.scala

```diff
@@ -70,11 +107,35 @@ final case class CreateAndExerciseCommand(
     choiceArgument: Value,
 ) extends ApiCommand
 
+/** Fetch either a template or interface */
 final case class FetchCommand(
     templateId: Identifier,
     coid: Value.ContractId,
 ) extends Command
```

Comment:
No need for this one. 
This is used only by canton for replay. They have the info need. 

Only the `APICommand` needs to be generic.

Reply:
Should I remove `FetchInterface` below as well? Doesn't sound useful by the same reason.

Reply:
And `ExerciseInterfaceCommand` as well.

Reply:
I dropped them.